src/Apps/Order/Routes/__tests__/Payment.jest.tsx

```diff
@@ -354,12 +355,14 @@ describe("Payment", () => {
     })
 
     it("renders bank element when bank transfer is chosen as payment method", async () => {
-      page.selectPaymentMethod(3)
+      page.selectPaymentMethod(0)
```

Comment:
**suggestion (non-blocking):** This is out of scope for this ticket, but it would be nice to ask for the specific payment method that you need without having to remember its index. If you were to add a `data-test-id` for each payment method, we can refactor this method to select the specific payment method that we want to test.